styles.css

```diff
@@ -1,23 +1,19 @@
-
 body {
-  background: #4ea0e8;
   border: 1px solid black;
+  background-image: url("https://media.freestocktextures.com/cache/bc/fc/bcfc7b6694542e0da1a6562ee676301e.jpg");
 }
 header {
-  max-width: 200px
-}
-
-.clear {
-  float:clear
+  max-width: 500px;
+  border:
 }
-img {
+.headerimage {
```

Comment:
Consider how you might reference an `img` in `header` without applying a `class` to it.